src/test/java/com/google/api/codegen/grpcmetadatagen/testdata/api_defaults.yaml

```diff
@@ -30,6 +30,6 @@ semver:
   #       publishing some 1.0.x versions to pypi already. They are now
   #       deleted but files still exist on the pypi servers and can prevent
   #       us from using the version number again.
-  python: '0.13.0'
```

Comment:
Actually, it ended up working kind of well, because I caught a change where I inaccurately bumped `oauthclient`.